pythainlp/util/__init__.py

```diff
@@ -103,6 +103,9 @@
     remove_zw,
     reorder_vowels,
 )
+from pythainlp.util.remove_trailing_repeat_consonants import (
+    remove_trailing_repeat_consonants,
```

Comment:
I think you also need to add `remove_trailing_repeat_consonants` name to `__all__` (at the very top of the file) as well.

Names in `__all__` will be exported when `from <module> import *` is used.

This is why codeacy/pyflakes said the function is unused -- as it is imported but has never been used/exported.

Reply:
Oh, I didn’t know that feature…
Fixed!

Reply:
Thank you.
(I never know it either until I start to maintain this lib lol.)